### ISSUE TITLE
Endure compatibility

### ISSUE THREAD

**Issue by xJoelituh** [Zarel/Pokemon-Showdown#1840]
So, if i'm right it says in PS! that Endure it's an actual Move tutor from ORAS, which is not. The ORAS metagame was introduced before the games were released with the data exported from the demo so I think that's the reason of that being incorrect. (http://www.smogon.com/forums/threads/oras-research-thread.3519780/ Concret pastebin: http://pastebin.com/8rbu6Fmp).

It was a TM in Gen 2, a Move Tutor in Gen 3 and again a TM in Gen 4. There isn't a legal form to catch/breed a Pokemon with a Hidden Ability and Endure(Unless it learns it by level up or Egg move, but these aren't all 721 Pokemon)

Here's a gallery made by myself with all the Tutors from ORAS that are in the Battle Resort where the 95% of the tutors are. That 5% are: Draco Meteor, Dragon Ascent, the FWG Pludges, Frenzy Plant, Blast Burn, Hydro Cannon, Relic Song and Secret Sword. 

http://imgur.com/a/Umaqi

To sum up: A lot of Pokemon in Gen 6 doesn't learn Endure, and also Endure is incomptible with the Hidden Abilities and Gen 5/6 breedings (Like Azumarill Belly Drum + Aqua Jet can't get Endure because it's "new" from XY, for the breeding changes.)

(A lot of lists that says which ones are the Move tutors: http://www.smogon.com/forums/threads/a-quickstart-guide-to-oras-breeding-training.3522921/ http://www.serebii.net/omegarubyalphasapphire/movetutor.shtml http://bulbapedia.bulbagarden.net/wiki/Endure_(move) http://www.ign.com/wikis/pokemon-omega-ruby-alpha-sapphire/Move_Tutors )

**Comment by TheImmortal** [comment]
The learnsets are obtained from the game. Tagging @Marty-D.

**Comment by Marty-D** [comment]
@TheImmortal, I'm not really sure what happened but I assume the list you originally used was the [first version that was crowdsourced](http://pastebin.com/zwG6puPT) (since the tutors were not stored with the move IDs). Apparently people thought Sleep Talk was Endure, which is an alright guess because Sleep Talk is a TM again, but they also made a similar mistake with Air Cutter actually being Roost.

In reality the tutors were mostly copied from B2W2 and Sleep Talk, Dark Pulse, and Roost were simply not used in ORAS.

**Comment by TheImmortal** [comment]
Probably that version, yeah. So, Endure should be removed as a 6T move? And, should something else be changed? Sleep Talk?

**Comment by Marty-D** [comment]
Yeah just 6T Endure is a mistake right now. It actually being Sleep Talk is irrelevant since it's not mapped to a tutor in-game; the Air Cutter thing was fixed last time, so I think we're good.
